gelato-web3-script/src/web3-functions/GearboxClaimAndBribe/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const GEARBOX_TREE_ADDRESS =
```

Comment:
In the end it would be good if this script was versitile/there were not fixed constants.  Maybe in the case of the tree claim it's just not doable, but we should try as much as we can.

I think you can pull the gearbox_tree_address from the contract.